src/tab/ExNavigationTabBar.js

```diff
@@ -6,6 +6,9 @@ import {
   TouchableNativeFeedback,
   Text,
   View,
+  Platform,
```

Comment:
Just noticed the unused variables here also.  Sorry for not catching the first time